utilities/check_mrTrix_Version.m

```diff
@@ -0,0 +1,19 @@
+function mrTrixVersion = check_mrTrix_Version
```

Comment:
This function needs to call AFQ_mrtrix_set_ld_path. Otherwise mrview will error because of libraries